src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -83,11 +76,21 @@ export function fetchCNPPaymentInformation(recordEvent = recordAnalyticsEvent) {
 
     if (response.error) {
       recordEvent({ event: `profile-get-cnp-direct-deposit-failed` });
+
+      captureCNPError(response, {
+        eventName: 'cnp-get-direct-deposit-failed',
+        useLighthouseDirectDepositEndpoint,
+      });
+
```

Comment:
This is the new capturing of error response and sending to sentry, just updates the event name, indicates if it was from lighthouse or legacy api, and then logs the error response, instead of trying to use a formatting function to pluck data from the error response.